--- eBaySvc.wsdl.org	2005-11-07 22:41:23.000000000 +0900
+++ eBaySvc.wsdl	2005-11-07 22:41:06.000000000 +0900
@@ -4579,5 +4579,5 @@ items at once.
 		</xs:annotation>
 		<xs:complexContent>
-			<xs:extension base="AbstractRequestType">
+			<xs:extension base="ns:AbstractRequestType">
 				<xs:sequence>
 					<xs:element name="Sort" type="ItemSortTypeCodeType" minOccurs="0">
@@ -4619,5 +4619,5 @@ items at once.
 		</xs:annotation>
 		<xs:complexContent>
-			<xs:extension base="AbstractResponseType">
+			<xs:extension base="ns:AbstractResponseType">
 				<xs:sequence>
 					<xs:element name="ItemsAwaitingFeedback" type="PaginatedTransactionArrayType" minOccurs="0">
@@ -5212,5 +5212,5 @@ property of the request object.
 		</xs:annotation>
 		<xs:complexContent>
-			<xs:extension base="AbstractRequestType">
+			<xs:extension base="ns:AbstractRequestType">
 				<xs:sequence>
 					<xs:element name="WatchList" type="ItemListCustomizationType" minOccurs="0">
@@ -5341,5 +5341,5 @@ property of the request object.
 		</xs:annotation>
 		<xs:complexContent>
-			<xs:extension base="AbstractResponseType">
+			<xs:extension base="ns:AbstractResponseType">
 				<xs:sequence>
 					<xs:element name="BuyingSummary" type="BuyingSummaryType" minOccurs="0">
@@ -5476,5 +5476,5 @@ property of the request object.
 	     </xs:annotation>
 		<xs:complexContent>
-			<xs:extension base="AbstractRequestType">
+			<xs:extension base="ns:AbstractRequestType">
 				<xs:sequence>
 					<xs:element name="BuyingReminders" type="ReminderCustomizationType" minOccurs="0">
@@ -5512,5 +5512,5 @@ property of the request object.
 	     </xs:annotation>
 		<xs:complexContent>
-			<xs:extension base="AbstractResponseType">
+			<xs:extension base="ns:AbstractResponseType">
 				<xs:sequence>
 					<xs:element name="BuyingReminders" type="RemindersType" minOccurs="0">
@@ -5554,5 +5554,5 @@ property of the request object.
 	      </xs:annotation>
 		<xs:complexContent>
-			<xs:extension base="AbstractRequestType">
+			<xs:extension base="ns:AbstractRequestType">
 				<xs:sequence>
 					<xs:element name="ScheduledList" type="ItemListCustomizationType" minOccurs="0">
@@ -5636,5 +5636,5 @@ property of the request object.
   		</xs:annotation>
 		<xs:complexContent>
-			<xs:extension base="AbstractResponseType">
+			<xs:extension base="ns:AbstractResponseType">
 				<xs:sequence>
 					<xs:element name="SellingSummary" type="SellingSummaryType" minOccurs="0">
